docs/install/binary.mdx

```diff
@@ -86,3 +87,23 @@ Below is an example:
   };
 }
 ```
+
+### Arch Linux
+
+The recommended way to install Ghostty is to install the existing package in the
+`extra` repository. This package is maintained by the community.
+
+```sh
+yay -S ghostty
```

Comment:
I know it’s already merged but it might be better to use `pacman` here rather than `yay` since the ghostty package isn’t in the AUR.

Reply:
What's the pacman command? The same? I can update it. Thanks!

Reply:
Yep!
```suggestion
pacman -S ghostty
```